R/checks.R

```diff
@@ -252,14 +252,41 @@ check_match_type <- function(x, y, by, match_type, verbose) {
 
   # Error if user chooses "1" but actually "m" ----
   if (match_type_error) {
+
     msg     <- "match type inconsistency"
     hint    <-
-      "you could use `return_report = TRUE` in `joyn::is_id()`
-    to see where the problem is"
+      "refer to the duplicate counts in the table(s) above
+       to identify where the issue occurred"
     joyn_msg("err")
+
+    display_id_x <- display_id_y <- NULL
+
+    # if x is not id (i.e., if xm = TRUE and user chooses 1:..)
+    if (x_m & tx == 1) {
```

Comment:
This condition is incorrect. if `tx == 1` is true, `x_m` is never evaluated, but it is TRUE by default in line 236.